plugins/namecolors-paper/src/main/java/com/biggestnerd/namecolors/NameColorSetting.java

```diff
@@ -123,33 +105,132 @@ public void clicked(Player p) {
         view.showScreen();
     }
 
+    public Clickable getRGBButton(MenuSection menu) {
+        Clickable button;
+        ItemStack is = new ItemStack(Material.WRITABLE_BOOK);
+        ItemUtils.setComponentDisplayName(is, MiniMessage.miniMessage().deserialize("Change to color of your name to any RGB value"));
+        button = new Clickable(is) {
+
+            @Override
+            public void clicked(Player target) {
+                new MenuDialog(target, NameColors.getInstance().setting, menu, "Please input a valid hex color") {
+                    @Override
+                    public void onReply(String[] message) {
+                        if (message.length > 1) {
+                            target.sendRichMessage("<red>RGB values cannot have spaces.</red>");
+                            return;
+                        }
+                        String result = message[0];
+                        if (!isValidValue(result)) {
```

Comment:
I don't know whether it does or not, but this should handle values that start with `#` as well as ones that don't.